.env

```diff
@@ -0,0 +1,6 @@
+DATABASE_USER=postgres
```

Comment:
Please add .env to the .gitignore file - usually .env has 'secrets' pr aPI keys, passwords, etc that should be shared in a public repo